src/vs/workbench/contrib/terminal/browser/media/shellIntegration.ps1

```diff
@@ -196,6 +196,13 @@ function Set-MappedKeyHandlers {
 			$result += "`a"
 			Write-Host -NoNewLine $result
 		}
+
+		Set-PSReadLineKeyHandler -Chord 'F12,g' -ScriptBlock {
+			Import-Module "$PSScriptRoot\GitTabExpansion.psm1"
```

Comment:
Do you need to import this over and over again?

It'll hit a cache and do nothing... but maybe you could just unregister the key handler

Reply:
It's tracked on the client side to make sure it only happens once but unregistering is a great idea